RestaurantManagementSystem/RestaurantManagementSystem/src/UseCases/GetRegistrationRequests/GetRequestsHandler.cs

```diff
@@ -0,0 +1,20 @@
+﻿using RestaurantManagementSystem.src.Core.Contracts;
+using RestaurantManagementSystem.src.Core.Entities;
+using RestaurantManagementSystem.src.Infrastructure.Db;
+
+namespace RestaurantManagementSystem.src.UseCases.GetRegistrationRequests
+{
+    public class GetRequestsHandler : IGetRequestsHandler
+    {
+        private readonly IRegistrationRequestRepo _registrationrequestrepo;
+        public GetRequestsHandler(IRegistrationRequestRepo registrationrequestrepo) {
+            _registrationrequestrepo = registrationrequestrepo;
+        }
+
+        public async Task<List<RegistrationRequest>> GetRequests()
```

Comment:
you should not return an entity from the use case there should be some kind of mapping, you shouldn't return the verification code for example.